packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -20715,6 +20725,51 @@ export default function MyFlexUpdateForm(props) {
           {...getOverrideProps(overrides, \\"Customtags\\")}
         ></TextField>
       </ArrayField>
+      <Autocomplete
+        label=\\"Label\\"
```

Comment:
Would be great if you could add an integ test for this work. Or cut a ticket tracking that.
Would be a simple non-DS backed form.
Also, I don't think the label's gonna show up here. I guess speaking to your previous work, `labelHidden=false` should be conditionally added to Autocompletes that are not `shouldWrapInArrayField`?

Reply:
Thanks
1.  Added cypress test for non array autocomplete field. 
2. Yeah, label wont show up in this PR. Ill rev the label PR with the conditional